ts/Core/Series/Point.ts

```diff
@@ -1009,19 +1011,21 @@ class Point {
                 }
             }
 
-            // record changes in the parallel arrays
-            i = point.index as any;
-            series.updateParallelArrays(point, i);
-
-            // Record the options to options.data. If the old or the new config
-            // is an object, use point options, otherwise use raw options
-            // (#4701, #4916).
-            (seriesOptions.data as any)[i] = (
-                isObject((seriesOptions.data as any)[i], true) ||
-                    isObject(options, true)
-            ) ?
-                point.options :
-                pick(options, (seriesOptions.data as any)[i]);
+            if (!(point as any).isNode) { // #11712
+                // record changes in the parallel arrays
+                i = point.index as any;
+                series.updateParallelArrays(point, i);
+
+                // Record the options to options.data. If the old or the new
+                // config is an object, use point options, otherwise use raw
+                // options (#4701, #4916).
+                (seriesOptions.data as any)[i] = (
+                    isObject((seriesOptions.data as any)[i], true) ||
+                        isObject(options, true)
+                ) ?
+                    point.options :
+                    pick(options, (seriesOptions.data as any)[i]);
+            }
```

Comment:
There's no need for `else`? Like, which arrays (perhaps `options.nodes`?) should be updated when updating a node? Is there a chance to move this "else" logic to NodesMixin? 🤔 It seems to be tricky...

Reply:
Yes its probably needed for exporting, maybe it will be dealt with by #15904?